### ISSUE TITLE
Question about current reccomended learning rates

### ISSUE THREAD

**Issue by wahrwolf** [OpenNMT/OpenNMT-py#1519]
Hi there,
according to the docs, the recommend learning rate for SGD is set to 1 (and it works quite well from my tests).

However this contradicts my understanding of the learning rate, since a value of 1 should not lead to any successful results.

Is the displayed learning rate somehow modified (e.g. log notation) or do I miss any "hidden parameter" here?

Also the recommended configs should be backed by a paper (if possible).

Thanks for any insights!

**Comment by vince62s** [comment]
what makes you say 1 should not lead to any successful results ?
anyway, SGD is less used nowadays, most people use adam, or similar optimizer for NMT tasks.

**Comment by wahrwolf** [comment]
I am still in university and my previous experiments where super bad (or we where advised to never use SGD) with high learning rates due to the high fluctuation and over correction.
However I have little experience in neural machine translation and since my results are awesome, I probably just have to read up a little ^^
Since the params overview is a little cluttered e.g. It is IMHO not clear which defaults will be actually used (like the adam params, which are described but will only be used if optim is set to adam), I thought I maybe missed something.

Nevertheless I think it would be nice to have some references for the recommendations (since this is a scientific project).

**Comment by guillaumekln** [comment]
> Is the displayed learning rate somehow modified (e.g. log notation) or do I miss any "hidden parameter" here?

Gradient clipping should be taken into account (see `-max_grad_norm`, defaults to 5).

The default optimization parameters are coming from https://arxiv.org/abs/1508.04025.
